verifier/verifier.go

```diff
@@ -177,6 +178,17 @@ func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelop
 			return err
 		}
 
+		pluginVersion := metadata.Version
+
+		//checking if the plugin version is in valid semver format
+		if !semVerRegEx.MatchString(pluginVersion) {
```

Comment:
Can we split `getVerificationPluginMinVersion` function into two functions one that performs semver related validation (empty and invalid) and othe rthat perform extraction and then reuse them?

Reply:
Done